src/matrixvariates.jl

```diff
@@ -45,6 +45,42 @@ Return the mean matrix of `d`.
 """
 mean(d::MatrixDistribution)
 
+"""
+    var(d::MatrixDistribution)
+
+Compute the matrix of element-wise variances for distribution `d`.
+"""
+var(d::MatrixDistribution) = ((n, p) = size(d); [var(d, i, j) for i in 1:n, j in 1:p])
+
+"""
+    cov(d::MatrixDistribution)
+
+Compute the covariance matrix for `vec(X)`, where `X` is a random matrix with distribution `d`.
+"""
+function cov(d::MatrixDistribution, ::Val{true}=Val(true))
+    M = length(d)
+    V = zeros(partype(d), M, M)
+    iter = CartesianIndices(size(d))
+    for el1 = 1:M
+        for el2 = 1:el1
+            i, j = Tuple(iter[el1])
+            k, l = Tuple(iter[el2])
+            V[el1, el2] = cov(d, i, j, k, l)
+        end
+    end
+    return V + tril(V, -1)'
+end
+
+"""
+    cov(d::MatrixDistribution, flattened = Val(false))
+
+Compute the 4-dimensional array whose `(i, j, k, l)` element is `cov(X[i,j], X[k, l])`.
+"""
+function cov(d::MatrixDistribution, ::Val{false})
```

Comment:
Should we add a `Val{true}` that fallsback to the normal cov?

Reply:
Did I not already do that [here](https://github.com/johnczito/Distributions.jl/blob/03b54b0c171173faef20d3142944bb2fd8e5e526/src/matrixvariates.jl#L60)?

Reply:
You did indeed, I mis-parsed it